index.html

```diff
@@ -11,12 +11,12 @@
   <body>
     <header>
       <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
+      <button onclick="showMobileMenu();" type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
       <nav id="desktop-nav">
         <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#projects">Portfolio</a></li>
           <li><a class="nav-link" href="#about-me">About</a></li>
           <li>
             <a class="nav-link" href="#contact"
```

Comment:
The previous reviewer mentioned, that the mobile menu should disappear when the user clicks any menu item. However, the menu still does not disappear, it stays on the top of the page. 

- [ ]  Please kindly make the menu hide when user clicks  any menu item 👍🏻 

[POSSIBLE SOLUTION]: You can add a click event listener on any menu item and set a function using JavaScript that hides the entire mobile menu.